src/Esprima/ParserExtensions.cs

```diff
@@ -33,6 +33,7 @@ internal static ReadOnlySpan<char> Between(this string s, int start, int end)
         "undefined", "length", "object", "Object", "obj", "Array", "Math", "data", "done", "args", "arguments", "Symbol", "prototype",
         "options", "value", "name", "self", "key", "\"use strict\"", "use strict"
     )]
+    [MethodImpl((MethodImplOptions) 512)]
```

Comment:
nit: kinda cryptic. Conditional compilation or a constant named `AggressiveOptimization` would be great (or at least a comment... :)

Reply:
Moved to constant